opae.spec.fedora

```diff
@@ -326,6 +326,7 @@ done
 %{_bindir}/pac_hssi_config.py
 %{_bindir}/rtl_src_config
 %{_bindir}/vabtool
+%{_bindir}/mem_tg
```

Comment:
I think maybe your branch is out-of-date with respect to the current master.  Could you rebase it?  The structure of this file, in particular, has changed last Friday.